### ISSUE TITLE
ColorVariable linter should not report lint for non-color properties

### ISSUE THREAD

**Issue by mikeholm** [causes/scss-lint#351]
here's a test case for color_variable_spec.rb that should not fail but does.

```ruby
  context 'when a color literal is not used' do
    let(:css) { <<-CSS }
      p {
        z-index: 9000;
        transition-duration: 250ms;
        cursor: default;
      }
    CSS

    it { should_not report_lint }
  end
```

**Comment by sds** [comment]
Thanks for the bug report, @mikeholm!

Fixed in 25f9d7b9d9.
